media/stylus/components/compat-tables/bc-table.styl

```diff
@@ -0,0 +1,308 @@
+    }
+}
+
+/* first column, fixed at 200 */
+.bc-table thead td,
+.bc-table tbody th {
+    width: 200px;
+    min-width: 200px;
+    max-width: 200px;
+    overflow: hidden;
+}
+
+/* everything except first column */
+.bc-table thead th,
+.bc-table tbody td {
+    width: 10%; /* not actually 10% but triggers making all equal width */
+}
+
+/* second column */
+.bc-table thead td + th,
+.bc-table tbody th + td {
+    border-left: 0;
+}
+
+/* thicker border between mediums */
+.bc-mediums th + th,
+.bc-medium-border {
+    border-left: 4px solid $bc-color-border;
+}
+
+
+/* th
+----------------------------------------------------------------------  */
+
+
+for $num in (2..3) {
+    .bc-table thead th[colspan={'"' + $num + '"'}] {
+        width: (100 / $num)%;
+    }
+}
+
+.bc-table .bc-mediums th {
+    padding: 2px 0 5px 0;
+    border-bottom: 4px solid $bc-color-border;
+    font-size: 24px;
+}
+
+.bc-table .bc-browsers th {
+    border-top: 0;
+    padding: 3px 0;
+    font-size:18px;
+}
+
+.bc-table tbody th {
+    border-right: 4px solid $bc-color-border;
+    border-left: 0;
+    text-align: left;
+}
+
+
+/* td
+----------------------------------------------------------------------  */
+
+/* empty td in thead */
+.bc-table thead td {
+    border-top-color: transparent;
+    border-right: 4px solid $bc-color-border;
+    border-left: 0;
+}
+
+/* make outline more visible */
+.bc-table td:focus {
+
+    z-index: 1;
+}
+
+.bc-table td[tabindex] {
+    cursor: pointer;
+}
+
+.no-js .bc-table tbody td,
+.bc-table tbody td[tabindex] {
+    padding-top: 6px;
+    padding-bottom: 12px;
+}
+
+
+/* code
+----------------------------------------------------------------------  */
+.bc-table tbody th code {
+    white-space: nowrap;
+}
+
+
+/*
+tablet display
+- declarataions are mixins
+- mixins are included in media queries below
+----------------------------------------------------------------------  */
+
+bc-table-tablet-display-block() {
+
+    .bc-table thead,
+    .bc-table tbody,
+    .bc-table tr {
+        width: 100%;
+        display: block;
+        clear: left;
+    }
+
+    .bc-table th,
+    .bc-table td {
+        display: block;
+        float: left;
+    }
+
+    .bc-table tbody th + td {
+        clear: left;
+    }
+
+    .bc-table thead td {
+        display: none;
+    }
+
+    .bc-table tbody th {
+        width: 100%;
+        max-width: 100%;
+        border-right: 0;
+        padding-top: 10px;
+        padding-bottom: 0;
+    }
+}
+
+bc-table-tablet-display-flex() {
+    /*
+    Flexbox for those lucky few who can handle it
+    - we only needs this to make equal height columns in the tablet display view
+    - ie is not among those that can handle this flexbox code,
+      but it doesn't supoort @support so we need to include that in the MQ
+    - all browsers which support @support also use unprefixed syntax
+    ----------------------------------------------------------------------  */
+
+    .bc-table,
```

Comment:
Discord is pretty cool! 

@openjck These comments are appearing a few lines higher than they probably should. Considering it's source mapped that's still pretty cool.